include/seqan/graph_types/graph_impl_directed.h

```diff
@@ -74,12 +74,13 @@ class Graph<Directed<TCargo, TSpec> >
 
 //____________________________________________________________________________
 
-
-        Graph() {
+        Graph()
```

Comment:
Please replace tabs with 4 spaces. Here you can find some notes about setting up you editor: http://seqan.readthedocs.org/en/master/HowTo/EditorSetup.html?highlight=editor